gamemode/modules/fadmin/fadmin/cl_interface/cl_scoreboardPlayer.lua

```diff
@@ -129,16 +129,16 @@ function FAdmin.ScoreBoard.Player.Show(ply)
     FAdmin.ScoreBoard.Player.Controls.ButtonPanel:Clear()
     FAdmin.ScoreBoard.Player.Controls.ButtonPanel:DockMargin(5, 5, 5, 5)
 
-
     for _, v in ipairs(FAdmin.ScoreBoard.Player.ActionButtons) do
-        if v.Visible == true or (isfunction(v.Visible) and v.Visible(FAdmin.ScoreBoard.Player.Player) == true) then
```

Comment:
This changes behaviour. When Visible is a function that returns false, the condition would still be true after this change. That's a bug.

Reply:
Sorry, I didn't realize that.